CMakeLists_IOS.txt

```diff
@@ -0,0 +1,206 @@
+# This file is based off of the https://code.google.com/archive/p/ios-cmake/
```

Comment:
@fluffypony 
Not sure of best practice here. 
https://code.google.com/archive/p/ios-cmake/ is licensed under "New BSD License"
I've altered the file a bit to make it work with monero. Should i add Monero copyright to header?